index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>My First Personal Website</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="About">About</a>
```

Comment:
the "About" in the `href` attribute will not work. You need add a `#` later. Just let you know.